lib/active_record/acts_as/instance_methods.rb

```diff
@@ -83,6 +93,9 @@ def column_for_attribute(name)
         end
       end
 
+      # Rails 6 introduces the additional argument time, which allows the setup of time
```

Comment:
Whats the impact of adding it?
ref: https://github.com/chaadow/active_record-acts_as/commit/064f32a8ea52d101a77aa25bcc6bf0edb0711a0f

Reply:
this one, I deviated from the upstream repo (chaadow) since they want to allow anyone using their library to setup the time when doing the touch function. Actually this time parameter is there already in Rails 6, but since in our CM repo we only call `touch` without specifying the time (basically everytime we call this function, we want to update the column `updated_at` to the current time), so I don't include the time parameter into this function